neuralpredictors/layers/__init__.py

```diff
@@ -1,23 +1,2 @@
-import torch
-
-from torch import nn
-
-
-class Bias2DLayer(nn.Module):
-    def __init__(self, channels, initial=0, **kwargs):
-        super(Bias2DLayer, self).__init__(**kwargs)
-
-        self.bias = torch.nn.Parameter(torch.empty((1, channels, 1, 1)).fill_(initial))
-
-    def forward(self, x):
-        return x + self.bias
-
-
-class Scale2DLayer(nn.Module):
-    def __init__(self, channels, initial=1, **kwargs):
-        super(Scale2DLayer, self).__init__(**kwargs)
-
-        self.scale = torch.nn.Parameter(torch.empty((1, channels, 1, 1)).fill_(initial))
-
-    def forward(self, x):
-        return x * self.scale
+from .affine import Bias2DLayer, Scale2DLayer
+from .conv import DepthSeparableConv2d
```

Comment:
Since you touched this file, would you do the honor of adding a package level docstring :)

Reply:
similar to:
https://github.com/sinzlab/neuralpredictors/blob/main/neuralpredictors/training/__init__.py